ansible/roles/common_installs/tasks/main.yml

```diff
@@ -95,6 +95,13 @@
     - fabric
     - virtualenv
     - sh
+    - httplib2
+
+- name: Install elasticsearch python client
```

Comment:
nit we may want to move this to the elastic search role since we only need it for those machines